File: doc/src/scripts/KerbalAlarmClock.py
import krpc
conn = krpc.connect(name='Kerbal Alarm Clock Example')

alarm = conn.kerbal_alarm_clock.create_alarm(
    conn.kerbal_alarm_clock.AlarmType.raw,
    'My New Alarm',
    conn.space_center.ut+10)

alarm.notes = '10 seconds have now passed since the alarm was created.'
alarm.action = conn.kerbal_alarm_clock.AlarmAction.message_only

File: doc/src/scripts/KerbalAlarmClock.java
import java.io.IOException;

import krpc.client.Connection;
import krpc.client.RPCException;
import krpc.client.services.KerbalAlarmClock;
import krpc.client.services.KerbalAlarmClock.Alarm;
import krpc.client.services.KerbalAlarmClock.AlarmAction;
import krpc.client.services.KerbalAlarmClock.AlarmType;
import krpc.client.services.SpaceCenter;

public class KerbalAlarmClockExample {
    public static void main(String[] args) throws IOException, RPCException {
        Connection connection = Connection.newInstance("Kerbal Alarm Clock Example", "10.0.2.2");
        KerbalAlarmClock kac = KerbalAlarmClock.newInstance(connection);
        Alarm alarm = kac.createAlarm(AlarmType.RAW, "My New Alarm", SpaceCenter.newInstance(connection).getUT() + 10);
        alarm.setNotes("10 seconds have now passed since the alarm was created.");
        alarm.setAction(AlarmAction.MESSAGE_ONLY);
    }
}
